src/server.ts

```diff
@@ -1,11 +1,12 @@
 import App from '@/app';
 import AuthRoute from '@routes/auth.route';
+import GroupsRoute from './routes/group.route';
```

Comment:
This import should be @routes/group.route try to avoid relative imports as much as possible